src/mame/heathkit/tlb.cpp

```diff
@@ -59,6 +59,22 @@
 
 #include <algorithm>
 
+#define LOG_REG (1U << 1)    // Shows register setup
+#define LOG_FUNC (1U << 4)   // Function calls
+
+#define VERBOSE (LOG_REG | LOG_FUNC)
+
+#include "logmacro.h"
+
+#define LOGREG(...)        LOGMASKED(LOG_REG, __VA_ARGS__)
+#define LOGFUNC(...)       LOGMASKED(LOG_FUNC, __VA_ARGS__)
+
+#ifdef _MSC_VER
```

Comment:
This seems like something the core should provide in some way.

Reply:
I saw this done like this in several other files, I guess it could go in logmacro.h, but that would probably require a large number of files to be changed.